generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.html.ejs

```diff
@@ -48,7 +48,7 @@
                         <label class="form-control-label" for="newPassword" jhiTranslate="global.form.newpassword.label">New password</label>
                         <input type="password" class="form-control" id="newPassword" name="newPassword"
                                placeholder="{{ 'global.form.newpassword.placeholder' | translate }}"
-                               formControlName="newPassword" #newPassword>
+                               formControlName="newPassword" data-cy="newResetPasswordSelector" #newPassword>
```

Comment:
can we use the same as id here?

Reply:
Yes, we can :)
We will make a commit to put some consistence.